eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/ExternalDependencyExposedCheck.java

```diff
@@ -25,14 +25,16 @@
         "Class ''%s'', is a class from external dependency. You should not use it as a return or method argument type.";
 
     private static final Set<String> VALID_DEPENDENCY_PACKAGE_NAMES = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
-        "java", "com.azure", "reactor"
+        "java", "com.azure", "reactor", "io.netty.buffer.ByteBuf"
```

Comment:
This is no longer a `VALID_DEPENDENCY_PACKAGE_NAMES` variable, as it also includes class names.